release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -45,6 +45,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-8-rhcos-butane"]
+==== Butane config transpiler improves processing of machine configs
```

Comment:
"improves creation" or "eases creation"?

Reply:
Hm, is "simplifies creation" too much of a stretch? Going with "improves creation" for now.